CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# XXXX.XX
+
+## Added
```

Comment:
Please correct the title to `## Features`

Reply:
Done